templates/role.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "betydb.fullname" . }}
+  labels:
+    {{- include "betydb.labels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources: ["pods", "endpoints", "Services", "configmaps"]
+  verbs:
```

Comment:
do we need any permissions? I don't think right now BETY needs permissions to any of these endpoints services.

Reply:
Endpoint access added for BETY role as we're accessing them in ingress config and readiness probe.